go.mod

```diff
@@ -0,0 +1,18 @@
+module github.com/ONSdigital/go-ns
+
+go 1.13
```

Comment:
I think this should be set to 1.12 for now, but generally we should be upgrading to the latest version of go (Im actually using 1.13.7)

Reply:
With the move to modules we are moving to 1.13 anyway so this library should only be used by apps built with 1.13.  It therefore seems sensible to leave this as 1.13.